tests/test_modeling_common.py

```diff
@@ -224,6 +224,7 @@ def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
 
     def test_save_load(self):
```

Comment:
would it make sense to have a `decorator` for this? which would set before and unset after?

Reply:
sure!

Reply:
Thank you, this is addressed.